src/CONFIG.js

```diff
@@ -6,15 +6,30 @@ import {addTrailingForwardSlash} from './libs/Url';
 import CONST from './CONST';
 
 // Set default values to contributor friendly values to make development work out of the box without an .env file
+const ENVIRONMENT = lodashGet(Config, 'ENVIRONMENT', CONST.ENVIRONMENT.DEV);
 const expensifyCashURL = addTrailingForwardSlash(lodashGet(Config, 'EXPENSIFY_URL_CASH', 'https://expensify.cash/'));
 const expensifyURL = addTrailingForwardSlash(lodashGet(Config, 'EXPENSIFY_URL_COM', 'https://www.expensify.com/'));
 const ngrokURL = addTrailingForwardSlash(lodashGet(Config, 'NGROK_URL', ''));
+const secureNgrokURL = addTrailingForwardSlash(lodashGet(Config, 'SECURE_NGROK_URL', ''));
+const expensifyURLSecure = addTrailingForwardSlash(lodashGet(
+    Config, 'EXPENSIFY_URL_SECURE', 'https://secure.expensify.com/',
+));
 const useNgrok = lodashGet(Config, 'USE_NGROK', 'false') === 'true';
 const useWebProxy = lodashGet(Config, 'USE_WEB_PROXY', 'true') === 'true';
 const expensifyComWithProxy = getPlatform() === 'web' && useWebProxy ? '/' : expensifyURL;
-const secureURLRoot = addTrailingForwardSlash(lodashGet(
-    Config, 'EXPENSIFY_URL_SECURE', 'https://secure.expensify.com/',
-));
+
+// Throw errors on dev if config variables are not set correctly
+if (ENVIRONMENT === CONST.ENVIRONMENT.DEV) {
+    if (!useNgrok && expensifyURL.endsWith('dev') && !expensifyURLSecure.endsWith('dev')) {
```

Comment:
In my .env file, I have a trailing slash for both the `EXPENSIFY_URL_COM` and the `EXPENSIFY_URL_SECURE` values, so this logic doesn't throw the error if I have `EXPENSIFY_URL_COM=https://www.expensify.com.dev/` and `EXPENSIFY_URL_SECURE=https://secure.expensify.com/`

Reply:
🤦 fixed this up and changed it to an `String.includes()`